client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -55,10 +57,26 @@ class _ProtectYourselfState extends State<ProtectYourself> {
   }
 
   List<Widget> _buildCards() {
+    final TextStyle normalText = TextStyle(
+      color: Constants.textColor,
+      fontSize: 16 * MediaQuery.textScaleFactorOf(context),
+      height: 1.4,
+    );
+    final TextStyle boldText = normalText.copyWith(fontWeight: FontWeight.w700);
     return (_factContent?.items ?? []).map((fact) {
-      print("fact body = ${fact.body}, image = ${fact.imageName}");
       return _ProtectYourselfCard(
-        message: _message(fact.body),
+        message: Html(
```

Comment:
How will this handle non-`<b>` tags? (I think the ideal is that non-`<b>` tags are not rendered)

Reply:
The Html widget has defaults for many of the standard tags.  We can customize their appearance in the same way that we do for the `<b>` tag in the `customTextStyle`.  I guess if we had a need to "turn off" tags we could also do it there.

Reply:
I think it would be best to "turn off" all other tags if possible (other than a predefined set of tags) so we don't get any other unexpected behavior?

Reply:
FYI to myself: Looks like a UI concern rather than a security one given `a, abbr, acronym, address, article, aside, b, bdi, bdo, big, blockquote, body, br, caption, cite, code, data, dd, del, dfn, div, dl, dt, em, figcaption, figure, footer, h1, h2, h3, h4, h5, h6, header, hr, i, img, ins, kbd, li, main, mark, nav, noscript, ol, p, pre, q, rp, rt, ruby, s, samp, section, small, span, strike, strong, sub, sup, table, tbody, td, template, tfoot, th, thead, time, tr, tt, u, ul, var` therefore, I don't think we should put in that effort.  

The risk here is a ugly UI from content we approved to go out.